test/test_code_to_gast_loops.py

```diff
@@ -0,0 +1,123 @@
+import unittest2
+import sys
+ 
+sys.path.append('code_to_gast')
+sys.path.append('code_to_gast/assign')
+sys.path.append('code_to_gast/expression')
+sys.path.append('code_to_gast/conditional')
+sys.path.append('code_to_gast/loop')
+sys.path.append('code_to_gast/helpers')
+sys.path.append('code_to_gast/routers')
+
+import py_main
+import js_main
+
+
+class test_code_to_gast_loops(unittest2.TestCase):
```

Comment:
Do you think we should continue testing, gast_to_code and code_to_gast or just test end to end?

Reply:
I think we should continue. @Danish21 and I discussed on our 1:1 that when adding new languages it will make it a lot easier to have these tests as you just need to test from code to gast and back. Also, I think it is a good thing to get into the practice of doing and it helps my workflow even though it is time-consuming.